platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/FCMQueryController.java

```diff
@@ -48,18 +51,19 @@ class FCMQueryController {
     private final int numberOfThreads;
     private final Throttle throttle;
 
-    private static final SpeedometerMetric.Config FCM_QUERIES_ANSWERED_PER_SECOND_CONFIG = new SpeedometerMetric.Config(
-                    "FCM", "fcmQueriesAnsweredPerSecond")
-            .withDescription("number of FCM queries have been answered per second")
-            .withFormat(FORMAT_9_6);
     private final SpeedometerMetric fcmQueriesAnsweredPerSecond;
 
     public FCMQueryController(final FCMConfig.FCMQueryConfig fcmQueryConfig, final Platform platform) {
         this.platform = platform;
         this.numberOfThreads = fcmQueryConfig.getNumberOfThreads();
         this.throttle = new Throttle(fcmQueryConfig.getQps());
-        this.fcmQueriesAnsweredPerSecond =
-                platform.getContext().getMetrics().getOrCreate(FCM_QUERIES_ANSWERED_PER_SECOND_CONFIG);
+        final Configuration configuration = platform.getContext().getConfiguration();
```

Comment:
With the given pattern that you introduce the `MetricsConfig` is passed to the individual creation of each `Metric`. I would have expected that the `MetricsConfig` is initially passed only once to the `Metrics` class.